forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/ApplicationAuditListener.java

```diff
@@ -82,6 +82,9 @@ protected Application prepareApplicationAudit(DelegateExecution execution) {
                 submittedBy = "Anonymous-user";
```

Comment:
is this code useless then?

Reply:
no. Only if authentication is null  submittedBy will have "service-account-forms-flow-bpm". 

![image](https://user-images.githubusercontent.com/99173163/179213452-dc6144d6-290d-4ed7-834c-f8f0ac8187e2.png)

Reply:
ok. another suggestion is to use an enumerator or constant rather than directly naming the variable on both cases

Reply:
done